index.js

```diff
@@ -1029,6 +1043,11 @@ function buildArray (location, code, name, key = null) {
 
   code += `
     var l = obj.length
+    if (l && l >= 20000) {`
```

Comment:
I think this const can make a single definition, and add a comment for this, such as:
```js
// please see : https://github.com/fastify/fast-json-stringify/pull/402
const kSize = 20000
...
code += `
  var l = obj.length
  if (l && l >=${kSize}) {`
...
```